src/main/webapp/app/exercises/text/assess-new/conflicts/text-feedback-conflicts.component.ts

```diff
@@ -121,8 +120,12 @@ export class TextFeedbackConflictsComponent extends TextAssessmentBaseComponent
 
         this.conflictingSubmissions = conflictingTextSubmissions;
         this.prepareTextBlocksAndFeedbackFor(this.leftSubmission!, this.leftTextBlockRefs, this.leftUnusedTextBlockRefs);
+        refreshLatestResult(conflictingTextSubmissions);
+        refreshLatestResult([this.leftSubmission!]);
         this.leftTotalScore = this.computeTotalScore(getLatestSubmissionResult(this.leftSubmission)!.feedbacks!);
         this.setConflictingSubmission(0);
+        refreshLatestResult(conflictingTextSubmissions);
```

Comment:
why do we need it twice?

Reply:
During testing we discoverd that after `this.prepareTextBlocksAndFeedbackFor()` we need to refresh. Insiede of `this.setConflictingSubmission(0);` `this.prepareTextBlocksAndFeedbackFor()` is called again.
Since we have indiviudal refrehs now we can find out which are really necessary. I'll check it

Reply:
Turns out we don't need it after all, it was only necessary when we still had the map for some reason. I removed the unused updates